### PR TITLE
Include position in the modals and arrow like a context panel

### PR DESCRIPTION
**Pull request by apenalver2**
Include in the dialogs the option to be positioned, defining the "x" and "y" and also include styles for the arrow like the context panels
ISSUE #196

### CODE REVIEW

File: src/app/showcase/components/dialog/showcase-dialog.component.ts [hunk 1]
```diff
@@ -33,11 +34,21 @@ export class ShowcaseDialogComponent {
 		this.dialogService.showDialog(ShowcaseSplitDialog, parameters);
 	}
 
-	public showCalendarDialog() {
+	public showCalendarDialog(event) {
```
Comment (alfonsserra) [inline src/app/showcase/components/dialog/showcase-dialog.component.ts]: Do we need the event?

File: src/app/showcase/components/dialog/showcase-dialog.component.ts [hunk 2]
```diff
@@ -33,11 +34,21 @@ export class ShowcaseDialogComponent {
...
 		const parameters: CalendarDialogParameters = CalendarDialog.getParameters();
 		this.dialogService.showDialog(CalendarDialog, parameters);
 	}
 
+	public showCalendarWithPositionDialog(event) {
```
Comment (alfonsserra) [inline src/app/showcase/components/dialog/showcase-dialog.component.ts]: Do we need the event?

File: src/app/showcase/components/dialog/showcase-dialog.component.html [hunk 3]
```diff
@@ -1,23 +1,27 @@
...
-            (click)="showTwoTabsDialog();">Two tab rows
+            (click)="showTwoTabsDialog()">Two tab rows
     </button>
     <button type="button" class="btn mt-2 mr-1"
-            (click)="showTwoColumnsDialog();">Two columns
+            (click)="showTwoColumnsDialog()">Two columns
     </button>
     <button type="button" class="btn mt-2 mr-1"
-            (click)="showCalendarDialog();">Calendar
+            (click)="showCalendarDialog($event)">Calendar
```
Comment (alfonsserra) [inline src/app/showcase/components/dialog/showcase-dialog.component.html]: Do we need the event?

Comment (apenalver2) [reply src/app/showcase/components/dialog/showcase-dialog.component.html]: Not needed, removed

File: src/app/showcase/components/dialog/showcase-dialog.component.ts (hunk 2 above)
Comment (apenalver2) [reply src/app/showcase/components/dialog/showcase-dialog.component.ts]: Not needed, removed

File: src/app/showcase/components/dialog/showcase-dialog.component.ts (hunk 1 above)
Comment (apenalver2) [reply src/app/showcase/components/dialog/showcase-dialog.component.ts]: Not needed, removed

**Comment by apenalver2** [comment]
Thanks, I will merge code with master branch, because we need, but as soon as I can I will include documentation about the new feature.

**Pull request merged**